<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>801</width>
      <height>551</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="browserLayout"/>
   </widget>
   <widget class="QPushButton" name="btnTopOptions">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>10</y>
      <width>88</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>options</string>
    </property>
   </widget>
   <widget class="qui::UrlBar" name="urlBar">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>5</y>
      <width>501</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>qui::UrlBar</class>
   <extends>QLineEdit</extends>
   <header location="global">src/qt_code/qui/url_bar.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
